polkadot-js/frontier-rpc-types.ts

```diff
@@ -0,0 +1,176 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
```

Comment:
Does this have to be a typescript file ? 
Can't we have it as a json ? (Even if it is through generation, I'd prefer to give people a json file for them to include than a ts for then to execute)

Reply:
I can include a json file. Can I leave this here though, in case we want to generate another one?